frontend/public/.eslintrc

```diff
@@ -1,5 +1,6 @@
 {
-  "extends": "eslint-config-mitodl",
+  "extends": ["eslint-config-mitodl", "eslint-config-mitodl/flow", "eslint-config-mitodl/mocha"],
+  "plugins": ["import"],
```

Comment:
This repo was using v0.7 of eslint-config-mitodl. The currently published version is 2.x and includes a base config and flow + mocha specific configs, and a jest specific config.